docs/source/introduction.rst

```diff
@@ -2,43 +2,162 @@
+scheduling provides a substantial potential increase in performance over
+serial execution.
+
+Event system
+------------
+
+Hyperledger Sawtooth supports creating and broadcasting events. This allows
+applications to:
+
+    * Subscribe to events that occur related to the blockchain, such as a new
+      block being committed or switching to a new fork.
+
+    * Subscribe to application specific events defined by a transaction family.
+
+    * Relay information about the execution of a transaction back to clients
+      without storing that data in state.
+
+Subscriptions are submitted and serviced over a ZMQ Socket.
+
+Ethereum contract compatibility with Seth
+-----------------------------------------
+
+The Sawtooth-Ethereum integration project, Seth, extends the interoperability
+of the Sawtooth platform to Ethereum. EVM (Ethereum Virtual Machine) smart
+contracts can be deployed to Sawtooth using the Seth transaction family.
+
+Pluggable consensus algorithms
+------------------------------
+
+In a blockchain, consensus is the process of building agreement among a group
+of mutually distrusting participants. Algorithms for achieving consensus with
+arbitrary faults generally require some form of voting among a known set of
+participants. General approaches include Nakamoto-style consensus, which
+elects a leader through some form of lottery, and variants of the traditional
+`Byzantine Fault Tolerance (BFT)
+<https://en.wikipedia.org/wiki/Byzantine_fault_tolerance>`_
+algorithms, which use multiple rounds of explicit votes to achieve consensus.
+
+Sawtooth abstracts the core concepts of consensus and isolates consensus from
+transaction semantics. The interface supports plugging in various consensus
+implementations. More importantly, Sawtooth allows different types of
+consensus on the same blockchain. The application can choose at compile time
+or change on a running blockchain with a transaction.
+
+Sawtooth currently supports these consensus implementations:
+
+    * Proof of Elapsed Time (PoET), a Nakamoto-style consensus algorithm that is
+      designed to be a production-grade protocol capable of supporting large
+      network populations. PoET relies on secure instruction execution to
+      achieve the scaling benefits of a Nakamoto-style consensus algorithm
+      without the power consumption drawbacks of the Proof of Work algorithm.
+
+    * PoET simulator, which provides PoET-style
+      consensus on any type of hardware.
+
+    * Dev mode, a simplified random-leader algorithm that is useful for
+      developers and test networks that require only crash fault tolerance.
+
+Sample transaction families
+---------------------------
 
-Transaction Families
-====================
 In Sawtooth, the data model and transaction language are implemented
 in a *transaction family*. While we expect users to build custom transaction
 families that reflect the unique requirements of their ledgers, we provide
```

Comment:
[ SHAWN ] Delete Validator Registry from the list below. It's really PoET Validator Registry, and is very specific to PoET.

Reply:
Done